org.uqbar.project.wollok.tests/src/org/uqbar/project/wollok/tests/interpreter/NullTestCase.xtend

```diff
@@ -0,0 +1,60 @@
+package org.uqbar.project.wollok.tests.interpreter
+
+import org.junit.Test
+
+class NullTestCase extends AbstractWollokInterpreterTestCase {
+	
+	@Test
+	def void messageSentToNull() {
+		'''
+		assert.throwsException({ null.sayHi() })
+		assert.throwsException({ null.toString() })
+		'''.test
+	}
+
+	@Test
+	def void numberOperatorsSentToNull() {
+		'''
+		assert.throwsException({ null + 3 })
+		assert.throwsException({ null - 3 })
+		assert.throwsException({ null * 3 })
+		assert.throwsException({ null / 3 })
+		'''.test
+	}
+	
+	@Test
+	def void booleanOperatorsSentToNull() {
+		'''
+		assert.throwsException({ null || true })
+		assert.throwsException({ null && true })
+		'''.test
+	}
+	
+	@Test
+	def void equalEqualOperatorSentToNull() {
+		'''
+		var valorNulo
+		assert.notThat(null == 8)
```

Comment:
I would like to include also

assert.notThat(8 == null)
and in fact we should test with different kind of receivers.

Crazy idea just from the top of my head, it would be simpler if the interpreter detects hat anyNotNullObject == null is false and avoid sending a message. Maybe that could be the default behavior of == (in object class) and then you only should override equals:

class Object {
 method ==(other) { return other != null && self.equals(other) }
}